### DISCUSSION TITLE
Notes in food log

### DISCUSSION THREAD

**Discussion by EcruBlack** [jlai/fitness-dashboard#48] [Ideas]
One of the things I liked about Fitbit's food logging page was the notes field at the top of every "meal." I used it to record the time I ate (my schedule is chaotic so I can never keep track) and when I took my various meds/supplements. I'm really bad about forgetting them if I don't keep track. Would that be easy to add? Thanks! I appreciate this site SO much!

**Comment by jlai** [comment]
Unfortunately, the Fitbit API for 3rd parties doesn't have a way to save notes and since this website doesn't have a database / account system and relies entirely on the Fitbit API, we don't have a way to store things like notes.

**Comment by EcruBlack** [comment]
Ah, shoot. Well, thank you for getting back to me!



________________________________
